index.html

```diff
@@ -43,20 +48,205 @@ <h2 class="job-title">I'm a Software Developer</h2>
+          </p>
+          <span>
+            <button>html</button><button>bootstrap</button><button>Ruby</button>
+          </span>
+          <a href="#"><button>See Project</button></a>
+        </article>
+      </section>
+      <section class="about-me">
+        <h1>About Me</h1>
+        <p>
+          Hello I'm a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don't hestiate to
+          contact me.
+        </p>
+        <a href="#"><button type="submit">Get My Resume</button></a>
+        <figure class="fig2">
+          <img src="Group.png" alt="" />
+        </figure>
+        <div id="style1">
+          <ul class="title-style">
+            <li>Language</li>
+          </ul>
+          <ul class="list-style">
+            <li>JavaScript</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </div>
+        <div id="style2">
+          <ul class="title-style">
+            <li>Frameworks</li>
+          </ul>
+          <ul class="list-style">
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSPec</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+          </ul>
+        </div>
+        <div id="style3">
+          <ul class="title-style">
+            <li>Skills</li>
+          </ul>
+          <ul class="list-style">
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
+      </section>
+    </main>
+    <footer>
+      <section class="get-in-touch">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+      </section>
+      <form>
+        <div class="form-group">
+          <input type="text" placeholder="Full Name" />
+          <input type="email" placeholder="Email" />
+          <textarea cols="33" rows="3" placeholder="Enter text here"></textarea>
+        </div>
+        <button type="submit">Get in touch</button>
+        <nav class="contact-me">
+          <a class="logo1" href="https://github.com/amexabee"
+            ><i class="fa fa-github"></i
+          ></a>
+          <a
+            class="logo2"
+            href="https://www.linkedin.com/in/amanuel-galema-56900024a/"
+            ><i class="fa fa-linkedin"></i
+          ></a>
+          <a class="logo3" href="https://angel.co/"
+            ><i class="fa fa-angellist"></i
+          ></a>
+          <a class="logo4" href="https://twitter.com/fountfillwblood"
+            ><i class="fa fa-twitter"></i
+          ></a>
+          <a class="logo5" href="https://medium.com/"
+            ><i class="fa fa-medium"></i
+          ></a>
+        </nav>
+      </form>
+    </footer>
```

Comment:
- [ ] I know that you spent a lot of time doing this part 👏🏽, but it is not required yet. Please 🙏🏽 remove it.

Reply:
@amexabee asked: Which code is not required is it all of the footer?  
- Yes, please the requirement for this project is to complete the "Works" section and the "About my self" section. We will continue with the "Contact form" next week. 😃